app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<div class="col-lg-12">
+<div id="wrapper">
+<main id="content" class="clearfix">
+        <div class="col-lg-8 col-lg-offset-2">
+            </div>
+    
+    <div class="panel panel-primary">
+    <header class="panel-heading">
+        <div class="text-center xt-heading-top">
+            <a class="back-to-search" href="{{ path("autoedits") }}"">
+                <span class="glyphicon glyphicon-chevron-left"></span>
+                Back
+            </a>
+            {{ wiki_link("User:" ~ username, url, username) }} &middot; {{ wikiName }}
+        </div>
+    </header>
+    <div class="panel-body xt-panel-body">
+    		<p>
+				<a href="{{ linkWikiScript(url, "title=Special:Log&type=block&user=&year=&month=-1&tagfilter=&user=User:"~username) }} ">{{ msg("block_log") }}</a>
+				{% if isWMFLabs() %}
+				&middot;
+				<a href="//tools.wmflabs.org/guc/?user={{username}}" >Global user contributions</a>
+				&middot;
+				<a href="//meta.wikimedia.org/w/index.php?title=Special%3ACentralAuth&target={{username}}" >Global Account Manager</a>
+				{% endif %}
+			</p>
+
+			<div class="panel panel-default">
+				<div class="panel-heading">
+					<h4  class="topcaption" >Summary <span class="showhide" onclick="javascript:switchShow( 'summary', this )">[hide]</span></h4>
+				</div>
+				<div class="panel-body" id="summary">
+					<table class="leantable table-condensed  xt-table">
+						<tr><td>Starting date</td>		<td>{{ start }}</td></tr>
+						<tr><td>Ending date</td>		<td>{{ end }}</td></tr>	
+						<tr><td>(Semi-)automated edits</td>	<td class="tdnum" >{{ total_semi }}</td></tr>
+						<tr><td>Total edits</td>		<td class="tdnum" >{{ total }}</td></tr>
+						<tr><td>Percentage</td>	<td class="tdnum" >{{ total_pct|number_format(2) }}%</td></tr>
```

Comment:
For these unsortable tables (in the summary), we've been doing something like:
```
<div class="col-lg-6 stat-list clearfix">
    <table class="table"><tbody>
        <tr>
            <td>{{ msg('start') }}</td>
            <td>{{ start }}</td>
        </tr>
        ...
    </tbody></table>
</div>
```
`.stat-list` will make it mobile-friendly